packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesFileDownloader.m

```diff
@@ -84,11 +84,19 @@ - (void)downloadManifestFromURL:(NSURL *)url
                                                      timeoutInterval:EXUpdatesDefaultTimeoutInterval];
   [self _setManifestHTTPHeaderFields:request];
   [self _downloadDataWithRequest:request successBlock:^(NSData *data, NSURLResponse *response) {
-    NSAssert([response isKindOfClass:[NSHTTPURLResponse class]], @"response must be a NSHTTPURLResponse");
+    if(![response isKindOfClass:[NSHTTPURLResponse class]]){
+      errorBlock([NSError errorWithDomain:EXUpdatesFileDownloaderErrorDomain
+                                     code:1040
```

Comment:
1030 was used so just used 1040 and incremented. Is the `errorWithDomain` correct?

Reply:
yep, looks good!